pandas/core/frame.py

```diff
@@ -9347,8 +9347,12 @@ def pivot(
             on the rows and columns.
         dropna : bool, default True
             Do not include columns whose entries are all NaN. If True,
-            rows with a NaN value in any column will be omitted before
-            computing margins.
+
+            * rows with a NA value in any column will be omitted before computing
```

Comment:
nit: `an NA` value, not `a NA` value. This is because the letter `N` is spelled `en` which starts with a vowel.